src/Sylius/Behat/Context/Ui/CheckoutContext.php

```diff
@@ -1223,6 +1223,36 @@ public function iShouldNotBeAbleToCompleteTheShippingStep()
     }
 
     /**
+     * @When I do not select any payment method
+     */
+    public function iDoNotSelectAnyPaymentMethod()
+    {
+        // Intentionally left blank to fulfill context expectation
+    }
+
+    /**
+     * @Then I should not be able to complete the payment step
+     */
+    public function iShouldNotBeAbleToCompleteThePaymentStep()
+    {
+        Assert::true(
+            $this->selectPaymentPage->isNextStepButtonUnavailable(),
+           'The "next step" button should be disabled, but it does not.'
```

Comment:
`but it is not.` although I'd be in favour of dropping this part in every method as it doesn't bring any value. If the message is in an exception and it says `The "next step" button should be disabled.` it's quite obvious what happened without an overly explicit explanation.